newm-server/src/main/kotlin/io/newm/server/features/cardano/repo/CardanoRepositoryImpl.kt

```diff
@@ -212,6 +212,20 @@ internal class CardanoRepositoryImpl(
         return response.utxosList
     }
 
+    override suspend fun queryUtxosByOutputRef(
```

Comment:
I've confirmed that this approach isn't needed.

Reply:
Removed